packages/core/src/components/Table/TableBody/TableBody.tsx

```diff
@@ -38,7 +38,7 @@ export interface HvTableBodyProps
 
 const tableSectionContext = {
   type: "body",
-  filterClassName: "grid",
+  filterClassName: "_grid",
```

Comment:
Do we really did this change? I couldn't understand it.

Reply:
Yes, it's an internal class that we're using in `HvFocus` - it shouldn't have such a generic/common name, as it might lead to collisions in `className`s